core/modules/node/node.module

```diff
@@ -1488,6 +1490,12 @@ function node_ranking() {
       // The promote flag is either 0 or 1, which is automatically normalized.
       'score' => 'n.promote',
     ),
+    'title_match' => array(
+      'title' => t('Content title match'),
+      // If the keywords appear in the title, return 1, else 0.
+      'score' => "IF(CONCAT(',', title, ',') REGEXP ',(:words),') > 0, '1','0')",
```

Comment:
That's strange. This works locally, but not on the sandbox server. I wonder why? Both are running PHP 5.3. My local is running `MySQL, MariaDB, or equivalent version 5.6.35` and the sandbox is running `MySQL, MariaDB, or equivalent version 5.5.48-37.8`